docs/reference/objects/package/step.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: default
+title: Package Step
+parent: Package
```

Comment:
Could this sit at the top level so its clear this isn't accessed via the package drop?
```suggestion
parent: Objects
```